### PR TITLE
Updated readme and old 'ai pipeline' references

### PR DESCRIPTION
**Pull request by ashok133**
Updates:
 - Replaced 'ai pipeline' references from code
 - updated readme to improve messaging around tool's use

### CODE REVIEW

**Comment by ashok133** [comment]
@googlebot I fixed it.

File: ai_pipeline/templates/setup.py [hunk 1]
```diff
@@ -1,31 +0,0 @@
-# Copyright 2020 Google Inc. All Rights Reserved.
```
Comment (TheMichaelHu) [inline ai_pipeline/templates/setup.py]: Accidental deletion?

File: README.md [hunk 2]
```diff
@@ -87,4 +86,4 @@ as inputs in the config file.
 | keep_checkpoint_max | Number of model checkpoints to keep. |
 | log_step_count_steps | Number of steps to run before logging training performance. |
 | eval_steps | Number of steps to use to evaluate the model. |
-| early_stopping_steps | Number of steps with no loss decrease before stopping early. |
+| early_stopping_steps | Number of steps with no loss decrease before stopping early. |
```
Comment (TheMichaelHu) [inline README.md]: nit: added some whitespace accidentally

**Review by TheMichaelHu — CHANGES_REQUESTED** [review]
Thanks for doing this!
Looks like you deleted a template by accident, looks good otherwise I think. Big changes like this will be a lot less scary once we add CI/CD and proper test coverage which I will hopefully do this week.

File: README.md (hunk 2 above)
Comment (ashok133) [reply README.md]: ack, updating in next commit

File: README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 1. XGBoost (XGB)
 
-We will first only consider Kubeflow Pipelines (KFP) for orchestrating ML pipelines built using various Cloud AI Platform (CAIP) features. Orchestration using Cloud Composer (CC) may be in scope in the future.
-
-The full project plan can be found [here](https://docs.google.com/document/d/11-ljj4D3UT-_bOyFeN_L_uRXQkM9G10bte9jy1yfSYA/edit?ts=5df59215).
+The following backends are currently supported for model training:
+1. [Google Cloud AI Platform](https://cloud.google.com/ai-platform) 
+1. [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs) (managed Kubeflow Pipelines)
 
 ## Setup
```
Comment (TheMichaelHu) [inline README.md:13]: optional: It would be cool if we also told the user which APIs to enable and provide links.

Comment (ashok133) [reply README.md:13]: ack, will add in a new commit later.

File: ai_pipeline/templates/setup.py (hunk 1 above)
Comment (ashok133) [reply ai_pipeline/templates/setup.py]: The renaming messed it up and .gitignore held it back. Will fix in next commit.

Comment (TheMichaelHu) [reply ai_pipeline/templates/setup.py]: oooh, shoot. Can you make it so gitignore allows this setup.py file/

I think you add something like
`! ml_pipeline_gen/templates/setup.py` at the end.

File: README.md [hunk 4]
```diff
@@ -26,15 +26,16 @@ pip install -r requirements.txt
...
-Follow these
+Create a Kubeflow deployment using Cloud Marketplace. Follow these
 [instructions](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials)
 to give the Kubeflow instance access to GCP services.
 
-## CAIP Demo
+> A future release will automate provisioning of KFP clusters and incorporate K8s Workload Identity for auth. 
+
+## Cloud AI Platform Demo
 This demo uses the scikit-learn model in `examples/sklearn/user_model.py` to
```
Comment (TheMichaelHu) [inline README.md]: optional: would you also mind changing `user_model.py` to `sklearn_model.py`. We did the rename a while back and never editted the README. Same applies for the user_model in the KFP section.

Comment (ashok133) [reply README.md]: ack, will do.

File: .gitignore [hunk 5]
```diff
@@ -9,7 +9,6 @@ models/*
 dist/*
 *.tar.gz
 config.yaml
-setup.py
```
Comment (TheMichaelHu) [inline .gitignore:9]: can you keep this and just allow the setup template to be changed?

`! ml_pipeline_gen/templates/setup.py`

Comment (TheMichaelHu) [reply .gitignore:9]: We do want to ignore the setup.py that is generated by the template, which is why this line is needed.

Comment (ashok133) [reply .gitignore:9]: ack, updated in new commit

**Review by TheMichaelHu — APPROVED** [review]
LGTM 👍

**Pull request merged**